src/librustc_errors/lib.rs

```diff
@@ -337,6 +338,265 @@ struct HandlerInner {
     stashed_diagnostics: FxIndexMap<(Span, StashKey), Diagnostic>,
 }
 
+/// A helper trait to implement methods on HandlerInner.
```

Comment:
Would help readability to add some backticks where types, function & macro calls, etc. are referenced.